snuba/subscriptions/executor_consumer.py

```diff
@@ -307,7 +295,6 @@ def submit(self, message: Message[KafkaPayload]) -> None:
 
     def close(self) -> None:
         self.__closed = True
-        self.__next_step.close()
```

Comment:
This was wrong. The close method should not close the following ones as the partition revoke calls join which can cause messages to be submitted.